.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ on:
     - cron: '0 0 * * *'
 
 env:
-  FAF_DB_VERSION: v136
+  FAF_DB_VERSION: v137-RC1
```

Comment:
You will also want to bump the minimum required version in `server/db/__init__.py` otherwise the version check at startup will still be looking for the old version.

Reply:
And ideally we have a full release before merging this, although I'm not sure what your plan is in terms of the RC.

Reply:
if the PR is approved we can release v137 before merging